src/spec-node/devContainersSpecCLI.ts

```diff
@@ -193,7 +193,7 @@ async function provision({
 	const addRemoteEnvs = addRemoteEnv ? (Array.isArray(addRemoteEnv) ? addRemoteEnv as string[] : [addRemoteEnv]) : [];
 	const addCacheFroms = addCacheFrom ? (Array.isArray(addCacheFrom) ? addCacheFrom as string[] : [addCacheFrom]) : [];
 	const additionalFeatures = additionalFeaturesJson ? jsonc.parse(additionalFeaturesJson) as Record<string, string | boolean | Record<string, string | boolean>> : {};
-	const idLabels = idLabel ? (Array.isArray(idLabel) ? idLabel as string[] : [idLabel]) : getDefaultIdLabels(workspaceFolder!);
+	const providedIdLabels = idLabel ? Array.isArray(idLabel) ? idLabel as string[] : [idLabel] : undefined;
```

Comment:
Current description of `id-label` -
> If no --id-label is given, one will be inferred from the --workspace-folder path.

In what scenario would `providedIdLabels` be undefined? Won't it conflict with the description (that there will be some inference and something will be set for id-label). Or does inferring refer to a possibility of being `undefined`?